tools/topology/topology2/include/components/tdfb.conf

```diff
@@ -0,0 +1,174 @@
+#
+#
+# A generic TDFB component. All attributes defined herein are namespaced
+# by alsatplg to "Object.Widget.tdfb.attribute_name"
```

Comment:
minor: one explanation of what TDFB means would be nice in tdfb.conf (it is in the git commit and other places so not a big issue)

Reply:
I just updated it, better to have it in the class definition .conf and not just in the commit message.